app/models/registerable_edition.rb

```diff
@@ -14,7 +14,7 @@ def slug
     # to be consistent with Panopticon's slug format.
     panopticon_slug = Whitehall.url_maker.public_document_path(edition).sub(/\A\//, "")
     if edition.deleted?
-      panopticon_slug.sub!(%r{/deleted-([^/]+)$}, '/\1')
+      panopticon_slug.sub!(%r{deleted-([^/]+)$}, '\1')
```

Comment:
This leaves open the possibility for false-positives.  Maybe it should start with `(\A|/)` - ie anchored to either the beginning, or immediately after a `/`.